README.md

```diff
@@ -0,0 +1,68 @@
+# OFirehose
+
+Firehose for the federated social web
+
+## Installation
+
+### Prerequisites
+
+You'll need four things to get started:
+
+* node.js 4.x or higher
+* npm 1.1.0 or higher
```

Comment:
I'd change this to match https://pumpio.readthedocs.io/en/latest/installation/prerequisites.html. I checked when I was writing that page; 2.15.x is what ships with Node 4

Reply:
Agree